CHANGELOG.md

```diff
@@ -20,6 +20,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Add the ``configureActionButtons`` method to the AdminExtensionInterface
 - Add the ``configureBatchActions`` method to the AdminExtensionInterface
 - Added the `getAccessMapping` method to the AdminExtensionInterface
+- Added `AbstractAdmin::configureBatchActions` method
```

Comment:
why is this line in this diff? line endings?

could change the diff,that this line does not appear?